Lots of unpriv spec reviewing (#600)
riscv/riscv-cheri: src/cheri/riscv-unpriv-integration.adoc

The rules for modifying the behavior of such instructions are:

> * The result written to the target register is always unchanged.
> ** The lower XLEN bits are always written with the value from RV32I/RV64I.

Reviewer: ```suggestion
* The lower XLEN bits of the result written to the target register are always the same as the value from RV32I/RV64I.
```

The first bullet is not quite true since there are extra bits that are written, how about just merging them?
